src/test/java/com/easypost/AddressTest.java

```diff
@@ -105,7 +105,13 @@ public void testAll() throws EasyPostException{
      */
     @Test
     public void testCreateVerify() throws EasyPostException {
-        Address address = Address.create(Fixture.incorrectAddressToVerify());
+        List<String> verificationList = new ArrayList<>();
+        verificationList.add("delivery");
```

Comment:
Let's change this from `delivery` as a string to a boolean `true` so it matches our other libs. This way we can keep the assertions the same if ever the verification criteria changes in the future.

Reply:
got it